oidc-conformance-tests/configure_is.py

```diff
@@ -0,0 +1,307 @@
+import re
```

Comment:
let's add WSO2 license to the script files. 
https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/master/components/org.wso2.carbon.identity.oauth/pom.xml#L2